scripts/fnord-counter.mjs

```
@@ -0,0 +1,26 @@
+  const results = {};
+  for (const f of readdirSync(dir)) {
+    const text = readFileSync(join(dir, f), 'utf8');
+    const matches = text.match(/fnord/g);
+    results[f] = matches.length;
+  }
+  return results;
+}
+
+// Returns the top-N files by fnord count.
+export function topFnordFiles(counts, n) {
+  const limit = parseInt(process.env.FNORD_LIMIT);
```

ESLint

[error] 'process' is not defined.

(no-undef)